autorest/codegen/models/dictionary_schema.py

```diff
@@ -45,6 +45,7 @@ def get_python_type(self, namespace: Optional[str] = None) -> str:
 
     @classmethod
     def from_yaml(cls, yaml_data: Dict[str, str], **kwargs: Any) -> "DictionarySchema":
+        # pylint: disable=arguments-differ
```

Comment:
Why?

Reply:
My bad, forgot to remove the arguments-differ suppression after getting rid of the need